### PR TITLE
Add new seller fields and add new fields to order line item

### PR DESCRIPTION
**Pull request by SergeyPoluyanov95**

### CODE REVIEW

File: Api/Data/MarketplacerSellerInterface.php [hunk 1]
```diff
@@ -188,4 +190,34 @@ public function getSourceCode(): string;
...
+     * @return self
+     */
+    public function setBaseDomesticShippingFreeThreshold(float $value): self;
+
+    /**
+     * Set base international shipping cost applied at the Seller level
+     *
+     * @return float
+     */
+    public function getBaseDomesticShippingFreeThreshold(): float;
```
Comment (ducktyper) [inline Api/Data/MarketplacerSellerInterface.php]: It seems like the comment `international shipping cost` is incorrect. Isn't it `domestic shipping free threshold`?

File: Model/MarketplacerSeller.php [hunk 2]
```diff
@@ -275,4 +275,46 @@ public function setSourceCode(string $sourceCode): self
...
+
+    /**
+     * Set base international shipping cost applied at the Seller level
+     *
+     * @return float
+     */
+    public function getBaseDomesticShippingFreeThreshold(): float
+    {
+        return (float)$this->_getData(MarketplacerSellerInterface::BASE_DOMESTIC_SHIPPING_FREE_THRESHOLD);
+    }
```
Comment (ducktyper) [inline Model/MarketplacerSeller.php]: Same here. It seems like the comment `international shipping cost` is incorrect. Isn't it `domestic shipping free threshold`?

**Comment by ducktyper** [comment]
I left a few comments, other than that, the code looks ok to me.
I see the CI failed due to the last commit message containing lines having more than 100 characters. Shortening the line or breaking it into multiple lines will fix this issue.

File: Api/Data/MarketplacerSellerInterface.php (hunk 1 above)
Comment (SergeyPoluyanov95) [reply Api/Data/MarketplacerSellerInterface.php]: @ducktyper fixed

File: Model/MarketplacerSeller.php (hunk 2 above)
Comment (SergeyPoluyanov95) [reply Model/MarketplacerSeller.php]: @ducktyper fixed

**Review by ducktyper — APPROVED** [review]
Looks good to me

**Pull request merged**